packages/components/src/font-size-picker/stories/index.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import FontSizePicker from '../';
+
+export default { title: 'FontSizePicker', component: FontSizePicker };
+
+const FontSizePickerWithState = ( { withSlider, disableCustomFontSizes, ...props } ) => {
+	const [ fontSize, setFontSize ] = useState( 16 );
+	const fontSizes = [
+		{
+			name: 'Small',
+			slug: 'small',
+			size: 12,
+		},
+		{
+			name: 'Medium',
+			slug: 'medium',
```

Comment:
The FontSizePicker component uses "normal" as the reset value, so probably should use that text instead of Medium.

Reply:
Good catch, thank you.